theme/less/_navigation.less

```diff
@@ -93,13 +95,16 @@
 #main-navigation li.has-children > a:after
 {
   content: url('/images/arrow_bullet.png');
-  padding-left: 10px;
+  padding-left: 11px;
+  vertical-align: sub;
 }
 
-#main-navigation ul ul li.has-children > a:after
+#main-navigation ul ul li.has-children a:after
 {
   position: absolute;
   right: 10px;
+  top: 33%;
```

Comment:
`vertical-align:middle` or `vertical-align:sub` doesn't work on #main-navigation ul ul li.has-children a:after because it's positioned absolutely. So I added there top with percentage units.

Reply:
Right, what about using:
```
display: flex;
justify-content: space-between;
```

On `<a>` element instead?

Reply:
It seems `display: inline-block` on `#main-navigation ul ul a` solves this vertical height problem. No need to change line-height for `<a>` , I guess...

Reply:
What about the absolute positioning? Not sure how solution with `display:  inline-block ` can solve it, might be I didn't understand your comment. But feel free to apply your changes and we can revisit this.

Reply:
LGTM.